### PR TITLE
Create a design for Buttonlog

### LINKED ISSUE #10: Create a design for Buttonlog
Should be done after #9

### PR DESCRIPTION
**Pull request by naila135**
Resolve #10

### CODE REVIEW

File: Specification/Design/DesignerView/Iteration3.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+![image](https://user-images.githubusercontent.com/82474250/195091983-c84360d7-d6ca-4d10-a053-3f9f556afe70.png)
+
+
+Link for more details:
+https://www.figma.com/file/CCTH8dHGP8VzqF1XrrqAWu/ButtonLog-UI?node-id=4%3A267
+
+The result of this iteration was the approval of the result of the iteration as the final UI design of the project.
+
+
+The final design will be divided into source files for developers.
```
Comment (sahlet-official) [inline Specification/Design/DesignerView/Iteration3.md:14]: Please change it to:

The [approved design]_(https://github.com/sahlet-official/buttonlog/tree/develop/Specification/Design/ApprovedDesign/ApprovedDesign.md) will be divided into source files for developers.

File: Specification/Design/FinalDesign/FinalDesign.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+This file represents final result of creating UX/UI for button log aplication.
```
Comment (sahlet-official) [inline Specification/Design/FinalDesign/FinalDesign.md:1]: Please
Change the file name to ApprovedDesign.md
Change the folder name to ApprovedDesign

File: Specification/Design/FinalDesign/FinalDesign.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+This file represents final result of creating UX/UI for button log aplication. 
+
+Style: minimalism.
+Color Palette: white, black.
```
Comment (sahlet-official) [inline Specification/Design/FinalDesign/FinalDesign.md:4]: Maybe
Color Palette: white, black, gray.
?

File: Specification/Design/FinalDesign/FinalDesign.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Main Screen:
+
+![main](https://user-images.githubusercontent.com/82474250/195093685-a1d9441a-b9ac-4aa5-8cac-9ee848fc196a.png)
+
+About Screen:
+
+![about](https://user-images.githubusercontent.com/82474250/195093797-acf22e59-7c50-4eb8-948b-b9a4c5004ab1.png)
+
+Clear Log dialog:
```
Comment (sahlet-official) [inline Specification/Design/FinalDesign/FinalDesign.md:18]: Please change it to:

**Clear log confirmation window**:
Use
image

to make bold text

File: Specification/Design/FinalDesign/FinalDesign.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+This file represents final result of creating UX/UI for button log aplication. 
+
+Style: minimalism.
+Color Palette: white, black.
+
+Figma link for review: https://www.figma.com/file/CCTH8dHGP8VzqF1XrrqAWu/ButtonLog-UI?node-id=4%3A267
+
+Raw images for easier viewing:
+
+Main Screen:
```
Comment (sahlet-official) [inline Specification/Design/FinalDesign/FinalDesign.md:10]: Please change it to:

**Main screen**:
Use
image


to make bold text

File: Specification/Design/FinalDesign/FinalDesign.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+![main](https://user-images.githubusercontent.com/82474250/195093685-a1d9441a-b9ac-4aa5-8cac-9ee848fc196a.png)
+
+About Screen:
```
Comment (sahlet-official) [inline Specification/Design/FinalDesign/FinalDesign.md:14]: Please change it to:

**About app screen**:
Use
image

to make bold text

File: Specification/Design/DesignerView/DesignerView.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+ButtonLog UX/UI View by Designer
+
+Task: develop ux/ui for the button log application.
+
+To achieve the result, the following criteria were set for ux markup:
+- design should be user-oriented.
+- ux markup should have a minimum of screens.
+- design should be implementable for developers.
+
+Based on UX requirements and product owner requirements, the following specifications were developed for the UI interface:
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: done

File: Specification/Design/DesignerView/DesignerView.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+- style: minimalism.
+
+The UX scheme was based on the scheme from ProdOwnerView
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: ok, done

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 2 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: done

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 3 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: You are right. Done

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 4 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: Thanks for recomendations, done

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 6 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: done

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 5 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: done

File: Specification/Design/DesignerView/DesignerView.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
+ButtonLog UX/UI View by Designer
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md:1]: done

File: Specification/Design/DesignerView/DesignerView.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+Task: develop ux/ui for the button log application.
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md:3]: done

File: Specification/Design/DesignerView/DesignerView.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+The UX scheme was based on the scheme from ProdOwnerView
+
+After clarification of the task, the following changes were made to the customer's UX scheme:
+- the side drop-down menu was removed and replaced with a lower central menu
+
+After 1st iteration of Ui developing such changes were made:
+- color palette: white, black and shades.
+
+The development of the UI interface was divided into iterations, each of which will be accompanied by consultations with the product owner and development team.
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: done

File: Specification/Design/DesignerView/DesignerView.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+
+The results of each iteration will be contributed to the repository, accompanied by a link to the Figma document.
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: done

[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: done

File: Specification/Design/DesignerView/Iteration1.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+This document is the result of the first iteration of creating a UI interface for the ButtonLog application.
+
+As a result of the first iteration, the following design option was proposed:
+
+![Frame 1](https://user-images.githubusercontent.com/82474250/194753446-b9b26e65-d0d0-4ecb-ad2a-78d778b74107.png)
+
+
+Link for more details:
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/Iteration1.md]: done

File: Specification/Design/DesignerView/DesignerView.md [hunk 14]
```diff
@@ -0,0 +1,23 @@
+ButtonLog UX/UI View by Designer
+
+Task: develop ux/ui for the button log application.
+
+To achieve the result, the following criteria were set for ux markup:
+- design should be user-oriented.
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: done

File: Specification/Design/DesignerView/Iteration1.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
...
+https://www.figma.com/file/CCTH8dHGP8VzqF1XrrqAWu/ButtonLog-UI?node-id=0%3A1
+
+As a result, product owner made a number of clarifications about the UI interface:
+-The color scheme has been changed and is now black and white and its shades
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/Iteration1.md]: done

[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/Iteration1.md]: done

File: Specification/Design/DesignerView/Iteration3.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
+This document is the result of the third(last) iteration of creating a UI interface for the ButtonLog application.
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/Iteration3.md]: done

File: Specification/Design/DesignerView/Iteration3.md (hunk 1 above)
Comment (naila135) [reply Specification/Design/DesignerView/Iteration3.md]: done

**Review by Yura-0 — DISMISSED** [review]

File: Specification/Design/ApprovedDesign/ApprovedDesign.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
+This file represents final result of creating UX/UI for button log aplication. 
+
+Style: minimalism.
+Color Palette: white, black, gray.
```
Comment (sahlet-official) [inline Specification/Design/ApprovedDesign/ApprovedDesign.md:4]: I think it is better to say
"shades of gray" instead "gray"

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 5 above)
Comment (sahlet-official) [reply Specification/Design/FinalDesign/FinalDesign.md]: it hasn't been done yet
@naila135

note:
Do not make ":" bold.

Sorry for this minor correction.

**Review by sahlet-official — CHANGES_REQUESTED** [review]

File: Specification/Design/DesignerView/DesignerView.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: ahhh now I understand what you mean, done

File: Specification/Design/DesignerView/DesignerView.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: actually done

File: Specification/Design/DesignerView/DesignerView.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: done

[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: Can you tell me where it is not done, I looked through and could not find.

File: Specification/Design/DesignerView/DesignerView.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: forgot about ApprovedDesign, now done.

File: Specification/Design/DesignerView/DesignerView.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/DesignerView/DesignerView.md]: no problem, i fixed it

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 4 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: done

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 5 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: done, nvm

File: Specification/Design/FinalDesign/FinalDesign.md (hunk 6 above)
Comment (naila135) [reply Specification/Design/FinalDesign/FinalDesign.md]: done

File: Specification/Design/ApprovedDesign/ApprovedDesign.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
+This file represents final result of creating UX/UI for button log aplication.
```
[the remark this answers is not in the document]
Comment (naila135) [reply Specification/Design/ApprovedDesign/ApprovedDesign.md]: done

File: Specification/Design/ApprovedDesign/ApprovedDesign.md (hunk 17 above)
Comment (naila135) [reply Specification/Design/ApprovedDesign/ApprovedDesign.md]: ok, done

File: Specification/Design/DesignerView/DesignerView.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (sahlet-official) [reply Specification/Design/DesignerView/DesignerView.md]: it hasn't been at the moment of my previous review.
My comment was made at 7:17PM (GTM+2) 18.10.2022.
at this moment state of your branch related to the PR was this:
Follow the link and you will see, that changes have been done with commit at 10:31PM (GTM+2) 19.10.2022.
https://github.com/sahlet-official/buttonlog/pull/18/commits/d673382880931f3d8a15afd5ca6826535eda4a69

Please don't cheat on me.
You are lying in my face, I hate it.
Don't do that!!!!!!!!!!!!!

@naila135

File: Specification/Design/DesignerView/DesignerView.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (sahlet-official) [reply Specification/Design/DesignerView/DesignerView.md]: it hasn't been done yet
@naila135

(in other lists also)
(in other documents also)

look at DesignerView/Readme.md
and all other documentation

File: Specification/Design/DesignerView/DesignerView.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (sahlet-official) [reply Specification/Design/DesignerView/DesignerView.md]: it hasn't been done at the moment of my previous review.
My comment was made at 7:17PM (GTM+2) 18.10.2022.
at that moment state of your branch related to the PR was this:
Follow the link and you will see, that changes have been done with commit at 10:31PM (GTM+2) 19.10.2022.
https://github.com/sahlet-official/buttonlog/pull/18/commits/d673382880931f3d8a15afd5ca6826535eda4a69

Please don't cheat on me.
You are lying in my face, I don't like it.
Don't do that!!!!!!!!!!!!!

File: Specification/Design/DesignerView/Readme.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+- design should be user-oriented;
+- ux markup should have a minimum of screens;
+- design should be implementable for developers;
+
+Based on UX requirements and Product Owner requirements, the following specifications were developed for the UI interface:
+- style: minimalism.
+
+The UX scheme was based on the scheme from [ProdOwnerView.md](./Specification/Design/ProdOwnerView/ProdOwnerView.md)
+
+After [clarification](./Specification/Design/DesignerView/Iteration1.md) of the task, the following changes were made to the customer's UX scheme:
```
[the remark this answers is not in the document]
Comment (sahlet-official) [reply Specification/Design/DesignerView/Readme.md:15]: it hasn't been done yet
@naila135

**Review by sahlet-official — CHANGES_REQUESTED** [review]

**Review by sahlet-official — APPROVED** [review]

**Pull request merged**